mods/persistence/controllers/subsystems/mining.dm

```diff
@@ -53,32 +54,43 @@ SUBSYSTEM_DEF(mining)
 
 	var/regen_interval = 90		// How often in minutes to generate mining levels.
```

Comment:
The mines are hardly being emptied as it stands given reports, and given the lag caused at least doubling the time between regen's is valid.
`var/regen_interval = 180`       // How often in minutes to generate mining levels.